subsys/ipc/ipc_service/lib/ipc_icmsg_buf.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <string.h>
+#include <errno.h>
+#include <ipc/ipc_icmsg_buf.h>
+
+
+#define cache_invalidate(region, size) /* nop */
+#define cache_flush(region, size) /* nop */
```

Comment:
why is that?

Reply:
I'm planning to add cache management later on. As you could notice read/write functions are already prepared for that. We just don't have correct calls for that in Zephyr. At least i don't see them.

Reply:
yes, we have. See `sys_cache_data_range()`

Reply:
Pushed.